stash-origin/image-config.d/40-generate-auth-file.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-# Generate the Auth File
-/usr/libexec/xcache/authfile-update --origin
+# Generate the Authfiles and scitokens.conf file
+if supervisord_is_enabled stash-origin; then
+    /usr/libexec/xcache/authfile-update stash-origin
+fi
+if supervisord_is_enabled stash-origin-auth; then
+    /usr/libexec/xcache/authfile-update stash-origin-auth
+fi
```

Comment:
So just to confirm - it is ok to do an authfile-update on both stash-origin and stash-origin-auth if they are both enabled, right?

Reply:
Yep.